README.md

```diff
@@ -65,6 +65,12 @@ a. To run the tests in headed mode:
 yarn cy:open
 ```
 
+If you encounter issues with `project:setup` or `resources:create` tasks because of SSL or certificate errors when running the tests locally, try the following:
+
+```sh
+NODE_TLS_REJECT_UNAUTHORIZED=0 yarn cy:open
+```
```

Comment:
Nice!

Reply:
After the migration, we changed the NPM script from `cy:open` to `e2e:open`. I rebased this pull request onto #1504 and made the change there, as it appears to work in this case.